pkg/services/publicdashboards/commands/generate_datasources/generate.go

```diff
@@ -45,7 +40,7 @@ func GetCompatibleDatasources(baseUrl string) ([]string, error) {
 	// we only consider a datasource to be supported when alerting and backend are both true
 	var supported []string
 	for _, datasource := range datasources {
-		if datasource.Alerting && datasource.Backend && !unsupportedDataSourcesMap[datasource.Slug] {
+		if datasource.Alerting && datasource.Backend {
```

Comment:
I would hold this change until we have a confirmation about Splunk. Right now, it should be on the unsupported Map.